packages/contracts-bedrock/src/EAS/EAS.sol

```diff
@@ -80,7 +80,8 @@ contract EAS is IEAS, Semver, EIP712Verifier {
     uint256[MAX_GAP - 3] private __gap;
 
     /// @dev Creates a new EAS instance.
-    constructor() Semver(1, 0, 2) EIP712Verifier("EAS", "1.0.1") { }
+    /// @custom:semver 1.2.0
+    constructor() Semver(1, 2, 0) EIP1271Verifier("EAS", "1.0.1") { }
```

Comment:
This should be 1.1.0

Reply:
Bumping this version will affect the EIP712 domain, thus invalidate pending EIP712 delegated attestations. IMHO, it's ok (there should not be many of those + they are easily invalidated/expire anyway). I'll bump it regardless, but FYI.

Reply:
Please see https://github.com/ethereum-optimism/optimism/pull/6764

Reply:
What is your reasoning with bumping the EIP712 version? The semver and the EIP712 version could be decoupled. Our policy is that bytecode changes are minor bumps and different security models are major bumps. I was thinking that we just leave the EIP712 version as is for simplicity and then only bump it in the future if we explicitly want to invalidate signatures

Reply:
Oh, my bad. I thought that you meant to bump the EIP712 version, since the `1.2.0` change was made later by you directly. In this case, please ignore my PR and let's change the version to `1.1.0` here, while considering leaving the EIP712 version as `1.0.1` (even though, bumping it would be de-facto pretty harmless at this stage as well).

Reply:
Ah apologies, i was not clear in my comment, I just left it for myself because i accidentally bumped it twice, will push a fix

Reply:
Ah, gotcha. Better safe than sorry! :D